src/server/kiwix-serve.cpp

```diff
@@ -624,7 +626,7 @@ static struct MHD_Response* handle_default(RequestContext* request_context)
 
   bool deflated = request_context->acceptEncodingDeflate && compress_content(content, mimeType);
   return build_response(
-      content.data(), content.size(), "", mimeType, deflated, true);
+      content.data(), content.size(), "", mimeType, deflated, false);
```

Comment:
And here :)

Reply:
I think the default page should not be cached, b'se when one adds a new zim to the library, we want the welcome page to show the updates list, and not the cached site.

Reply:
You're totally right here. Good catch.